blocks/image-near-hotels/image-near-hotels.js

```diff
@@ -0,0 +1,33 @@
+function createImageTeaser(block) {
+  const rows = block.querySelectorAll('.image-near-hotels > div');
+  rows[1].style.display = 'none';
+  rows[1].className = '';
+
+  const linkInImage = rows[1].getElementsByTagName('div');
+  const divForLink = document.createElement('div');
+  divForLink.className = 'text-in-image';
+  divForLink.innerHTML = linkInImage[0].firstChild.outerHTML;
+
+  const arrowRight = document.createElement('span');
+  arrowRight.className = 'ch-icon-arrow-right';
+  arrowRight.innerHTML = '&nbsp;';
+
+  const textWithArrow = divForLink.innerHTML.trim();
+  const linkWithArrow = textWithArrow.slice(0, (textWithArrow.length) - 4)
+  + arrowRight.outerHTML + textWithArrow.slice((textWithArrow.length) - 4);
+  divForLink.innerHTML = linkWithArrow;
+  const picture = block.querySelectorAll('.image-near-hotels > div > div');
+  picture[0].appendChild(divForLink);
+
+  const mapMarker = document.createElement('span');
```

Comment:
i don't think you need an extra span for the map marker - you can prepend it directly to the text by using the before pseudo class:
```
.image-near-hotels.block > div:nth-child(3) > div:before {
    content: '';
    font-family: 'chFont';
}
```